scripts/bootstrap/compile.sh

```diff
@@ -16,8 +16,13 @@
 
 # Script for building bazel from scratch without bazel
 
-PROTO_FILES=$(ls src/main/protobuf/*.proto src/main/java/com/google/devtools/build/lib/buildeventstream/proto/*.proto src/main/java/com/google/devtools/build/skyframe/*.proto src/main/java/com/google/devtools/build/lib/skyframe/proto/*.proto)
-LIBRARY_JARS=$(find derived/jars third_party -name '*.jar' | grep -Fv JavaBuilder | grep -Fv third_party/guava | grep -Fv third_party/guava | grep -ve 'third_party/grpc/grpc.*jar' | tr "\n" " ")
+if [ -d derived/jars ] then
+  PROTOBUF_JARS=derived/jars
+else
+  PROTOBUF_JARS="/usr/share/java/protobuf-3.11.4.jar /usr/share/java/protobuf-java-util-3.11.4.jar"
```

Comment:
nit: can you extract the version into a variable?

Reply:
Sounds good!